lib/locales/en.ts

```diff
@@ -622,6 +623,7 @@ export const locale: Locale = {
     kilian_of_wurzburg_bishop_and_companions_martyrs: 'Saint Kilian, Bishop, and Companions, Martyrs',
     kinga_of_poland_virgin: 'Saint Kinga, Virgin',
     kuriakose_elias_of_the_holy_family_chavara_priest: 'Saint Kuriakose Elias of the Holy Family Chavara, priest',
+    labour_day: 'Labour Day',
```

Comment:
In `en.ts`, we should use American English. Override this in `en-gb.ts` to `Labour Day` please.

```suggestion
    labor_day: 'Labor Day',
```

Reply:
Indeed 👍

Reply:
Could you also define the key in `en-gb.ts` please? :wink:

Reply:
This goes beyond the scope of this pull request 😉, but ok, I've added the commit 😄.

Reply:
Thanks! :pray: